ci/buildkite-secondary.yml

```diff
@@ -7,6 +7,14 @@ steps:
     agents:
       queue: "release-build"
     timeout_in_minutes: 10
+  - name: "publish dry-run"
+    command: "ci/publish-crate.sh --dry-run"
+    timeout_in_minutes: 180
+    artifact_paths: "log-*.txt"
+    env:
+      CRATE_PUBLISH_TEST: "true"
+    agents:
+      queue: "release-build"
```

Comment:
I think this one should happen in the main pipeline or a different pipeline instead of here. some reasons:

1. I don't think we want the crates publishing issue will block our bin shipping
2. PRs won't trigger this pipeline, which means we won't be able to catch this error immediately

Reply:
okay. if we want to to do so, I will suggest that we can use `queue: "solana"` instead of `queue: "release-build"`.

Reply:
What is the benefit of moving it to the other queue?

I see there's only one agent in the `release-build` queue. Are you concerned about the check taking up too much time in that queue?